Source/automap.cpp

```diff
@@ -17,6 +17,24 @@ int AutoMapYPos; // weak
 int AMPlayerX; // weak
 int AMPlayerY; // weak
 
+// color used to draw the player's arrow
+#define COLOR_PLAYER      (PAL8_ORANGE+1)
+// color for bright map lines (doors, stairs etc.)
+#define COLOR_BRIGHT      PAL8_YELLOW
+// color for dim map lines/dots
+#define COLOR_DIM         (PAL16_YELLOW+8)
+
+#define MAPFLAG_TYPE      0x000F
+// these are in the second byte
+#define MAPFLAG_DOORR     0x01

```

Comment:
Consider renaming these to match the names derived from the debug release:

* HorzArch
* VertArch
* HorzDoor
* VertDoor
* HorzGrate
* VertGrate
* Square
* Stairs

Reply:
The OBJECT flags are the arches (small square), the HALF flags are grates (small square with an attached wall).
I think the L flags are horizontal and the R flags are vertical, based on the check in GetAutomapType that forces type 7 to type 1 for the intersection of two arches.